app/code/Magento/Tax/Pricing/Render/Adjustment.php

```diff
@@ -173,4 +175,17 @@ public function displayPriceExcludingTax()
     {
         return $this->taxHelper->displayPriceExcludingTax();
     }
+
+    /**
+     * Obtain a value for data-price-type attribute
+     *
+     * @return string
+     */
+    public function getDataPriceType()
+    {
+        if ($this->getData('price_type') && $this->getData('price_type') !== 'finalPrice') {
+            return 'base' . ucfirst($this->getData('price_type'));
+        }
+        return 'basePrice';
```

Comment:
```suggestion
        if ($this->amountRender->getPriceType() === 'finalPrice') {
            return 'basePrice';
        }
        return 'base' . ucfirst($this->amountRender->getPriceType());
```

I guess it may be easier to understand the `===` instead of `!==`. Also, it may be better to use the `$this->amountRender->getPriceType()` instead of the `getData`. What do you think?